.golangci.yml

```diff
@@ -0,0 +1,65 @@
+service:
+  golangci-lint-version: 1.23.6 # Keep this in sync with .circleci/config.yml and Jenkinsfile.pr
+
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  gofmt:
+    simplify: true
+  govet:
+    check-shadowing: false # set this to true from time to time to check for possible issues
+    disable-all: true
```

Comment:
Tested locally. Remove both `disable-all` and `enable` stanzas. That makes it run everything. Then toggle the `check-shadowing` switch.

Reply:
Actually I kind of like the opt in version. We always know what we are enabling and we do the same below for the linters.

Reply:
Fine by me. I was just referring to the offline discussion we were having.

Reply:
Of course. I just wasn't sure what was enabled by default that way since I wasn't getting any error. I think this approach is probably better in the end since it gives us a bit more control over what to check.